docker/linux/clang-3-9/Dockerfile

```diff
@@ -24,5 +24,12 @@ RUN apt update -qqy \
     && apt install -qqy --no-install-recommends git \
     && /opt/clean-after-apt.sh
 
+# === Install nasm 2.16 using backport from PPA
```

Comment:
Looks better, but now you can consolidate this block with the above that already does the other PPA and cleanup - less clutter and faster build ?

Reply:
Ack, condensed the RUN directives